From d79c6de44ea47bb3b5c6d2f1759c35944a2f0e91 Mon Sep 17 00:00:00 2001
From: Avi Drissman <avi@chromium.org>
Date: Wed, 13 Jul 2022 18:29:31 +0000
Subject: [PATCH 082/110] Disable failing test.

(cherry picked from commit e0f63b352039c6a59c2aa34d94f74c08d684651c)

Bug: 1198780
Change-Id: Ic5be6ec7e7e5ee4a1fa0e31b9cf9c8039bc4072b
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3738947
Reviewed-by: John Abd-El-Malek <jam@chromium.org>
Auto-Submit: Avi Drissman <avi@chromium.org>
Commit-Queue: John Abd-El-Malek <jam@chromium.org>
Commit-Queue: Avi Drissman <avi@chromium.org>
Cr-Original-Commit-Position: refs/heads/main@{#1020201}
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3751372
Reviewed-by: Evan Stade <estade@chromium.org>
Owners-Override: Joe Downing <joedow@chromium.org>
Auto-Submit: Robert Liao <robliao@chromium.org>
Commit-Queue: Joe Downing <joedow@chromium.org>
Reviewed-by: Joe Downing <joedow@chromium.org>
Owners-Override: Robert Liao <robliao@google.com>
Cr-Commit-Position: refs/branch-heads/5005@{#1241}
Cr-Branched-From: 5b4d9450fee01f821b6400e947b3839727643a71-refs/heads/main@{#992738}
---
 .../preinstalled_web_app_definition_utils_unittest.cc        | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/chrome/browser/web_applications/preinstalled_web_apps/preinstalled_web_app_definition_utils_unittest.cc b/chrome/browser/web_applications/preinstalled_web_apps/preinstalled_web_app_definition_utils_unittest.cc
index aae1ce25a6eb..e9b671654d3d 100644
--- a/chrome/browser/web_applications/preinstalled_web_apps/preinstalled_web_app_definition_utils_unittest.cc
+++ b/chrome/browser/web_applications/preinstalled_web_apps/preinstalled_web_app_definition_utils_unittest.cc
@@ -19,8 +19,9 @@ class PreinstalledWebAppUtilsTest : public testing::Test {
   ~PreinstalledWebAppUtilsTest() override = default;
 };
 
-// https://crbug.com/1198780 tracks test failures on Linux and ChromeOS.
-#if BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_CHROMEOS)
+// https://crbug.com/1198780 tracks test failures due to memory smashing on
+// Linux, ChromeOS, and the Mac.
+#if BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_CHROMEOS) || BUILDFLAG(IS_MAC)
 #define MAYBE_GetTranslatedName DISABLED_GetTranslatedName
 #else
 #define MAYBE_GetTranslatedName GetTranslatedName
-- 
2.38.1

